physionet-django/events/templates/events/event_home.html

```diff
@@ -11,193 +11,255 @@
+          participants</button>
+        <a class="btn btn-sm btn-secondary" href="{% url 'update_event' event.slug %}"
+          data-form-url="{% url 'update_event' event.slug %}">Edit Event</a>
+        {% endif %}
+
+        <a href="{% url 'event_detail' event.slug %}" class="btn btn-sm btn-info">View Event</a>
+      </li>
       {% empty %}
-        <div class="card-body text-center">
-          <p class="card-text">No events to show</p>
-        </div>
+      <div class="card-body text-center">
+        <p class="card-text">No events to show</p>
+      </div>
       {% endfor %}
     </ul>
   </div>
   <br>
   {% if events_past %}
-    {% for event in events_past %}
-    {% for info in event_details|get_applicant_info:event.id %}
-      <div class="modal fade" id="{{ info.id }}-modal-{{ event.id }}" tabindex="-1" role="dialog" aria-labelledby="{{ info.id }}-modal" aria-hidden="true">
-        <div class="modal-dialog modal-lg"  role="document">
-          <div class="modal-content">
-            <div class="modal-header">
-              <h5 class="modal-title">{{ info.title }}</h5>
-              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                <span aria-hidden="true">&times;</span>
-              </button>
-            </div>
-              {% include 'events/event_applications.html' %}
-            </div>
-          </div>
+  {% for event in events_past %}
+  {% for info in event_details|get_applicant_info:event.id %}
+  <div class="modal fade" id="{{ info.id }}-modal-{{ event.id }}" tabindex="-1" role="dialog"
+    aria-labelledby="{{ info.id }}-modal" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">{{ info.title }}</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
         </div>
-    {% endfor %}
-    {% endfor %}
-  {% endif %}
+        {% include 'events/event_applications.html' %}
+      </div>
+    </div>
   </div>
+  {% endfor %}
+  {% endfor %}
+  {% endif %}
+</div>
 {% endblock %}
 
-
 {% block local_js_bottom %}
-  <script>
-  </script>
-  <script src="{% static 'custom/js/resize-ck.js' %}"></script>
-{% endblock %}
+<script src="{% static 'custom/js/resize-ck.js' %}"></script>
+<script>
+  $(document).ready(function () {
+    $('.manage-cohost').submit(function (e) {
+      e.preventDefault();
+      let $form = $(this);
+      $form.find('input[type="submit"]').prop('disabled', true);
+
+      let participant_id = e.target.participant.value;
+      let event_slug = e.target.event.value;
+      let submit_value = $form.find('input[type="submit"]').val();
+      let action = $form.find('input[type="submit"]').attr('data-action');
+      let csrftoken = e.target.csrfmiddlewaretoken.value;
+      let url = e.target.action;
```

Comment:
We could always prefer `const` over `let`. It's always easier to reason about the code when you cannot change certain values.